content/posts/0004-writing-buildpacks.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Building a Cloud Native Buildpack Isn’t as Hard as You Think
+date: "2021-02-05"
+slug: building-a-cloud-native-buildpack-isnt-as-hard-as-you-think
+author: swigmore
+---
+
+Maybe you’ve heard some chatter about Cloud Native Buildpacks, or at least the value in containerizing your applications. Buildpacks provide a seamless mechanism for doing just that. There are a few implementations of the buildpack concept available, but the general takeaways can be summarized as follows:
+
+> “Buildpacks provide a seamless experience from source code to a running container” 
+
+> “Dockerfiles aren’t the only option anymore” 
+
+> “Everything is open source and community driven”
+
+The Paketo project provides CNB implementations for the most popular languages. Check out https://paketo.io/ to learn more about what's supported. Our buildpacks provide great out-of-box support for most languages, but what if I need a buildpack that doesn’t exist yet? As a community-driven project, is the onus on me to create that buildpack? Where do I even begin?
```

Comment:
This paragraph is a great intro into what the blog is going to be about. I think that you should make it more prominent in some way.